app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -60,7 +60,7 @@ angular.module('miq.util').factory('chartsMixin', function() {
   };
 
   return {
-    dashboardHeatmapChartHeight:    286,
+    dashboardHeatmapChartHeight:    281,
```

Comment:
Think of this as a style change. Had to make adjustments to make everything look good.